.gitignore

```diff
@@ -1,2 +1,3 @@
 Gemfile*
+up.sh
```

Comment:
Ahh, yes - I added it to .gitignore because I didn't want it to affect anything but you're right - wouldn't mean anything to anyone downloading this to work on themselves.